src/eva/core/data/datasets/embeddings/classification/multi_embeddings.py

```diff
@@ -0,0 +1,106 @@
+"""Dataset class for where a sample corresponds to multiple embeddings."""
+
+import os
+from typing import Callable, Dict, List, Literal
+
+import numpy as np
+import torch
+from typing_extensions import override
+
+from eva.core.data.datasets.embeddings import base
+
+
+class MultiEmbeddingsClassificationDataset(base.EmbeddingsDataset):
+    """Dataset class for where a sample corresponds to multiple embeddings.
+
+    Example Usecase: Slide level dataset where each slide has multiple patch embeddings.
+    """
+
+    def __init__(
+        self,
+        root: str,
+        manifest_file: str,
+        split: Literal["train", "val", "test"],
+        column_mapping: Dict[str, str] = base.default_column_mapping,
+        embeddings_transforms: Callable | None = None,
+        target_transforms: Callable | None = None,
+    ):
+        """Initialize dataset.
+
+        Expects a manifest file listing the paths of `.pt` files containing tensor embeddings.
+
+        The manifest is required to have a `column_mapping["multi_id"]` column that contains the
```

Comment:
"The manifest is required to" --> "The manifest file must"

Reply:
Done